ai-catalog/samples/gemini-chatbot/src/main/java/com/android/ai/samples/geminichatbot/GeminiChatbotScreen.kt

```diff
@@ -189,8 +187,10 @@ fun MessageBubble(
 }
 
 @Composable
-fun SeeCodeButton(context: Context) {
+fun SeeCodeButton() {
+    val context = LocalContext.current
```

Comment:
This usage of context is ok. Ideally you should be passing events up like onSeeCodeClicked to the activity where you could then launch, but it seems overkill just to open a link